src/content/en/updates/2017/12/modulepreload.md

```diff
@@ -0,0 +1,138 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: &lt;link rel="modulepreload"&gt; offers a way of declaratively loading ES2015 modules ahead of time.
```

Comment:
Should we drop the date? ES2015 -> JavaScript

Reply:
Done.